Homework2/Program.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System;
+
+namespace Homework2
+{
+    public class Program
+    {
+        private const string password = "SecretPassw0rd";
+        private const string username = "SecretUsEr";
+
+        public static void Main(string[] args)
+        {
+            bool loginSucceeded = false;
+            do
+            {
+                Console.WriteLine("Please enter your username:");
+                var user = Console.ReadLine();
+                Console.WriteLine("Please enter your password:");
+                var pw = Console.ReadLine();
+
+                loginSucceeded =
+                    string.Equals(user, username, StringComparison.OrdinalIgnoreCase) &&
+                    string.Equals(pw, password, StringComparison.Ordinal);
+            } while (!loginSucceeded);
```

Comment:
If you're going to loop in this way it is generally nice to show some kind of "username or password incorrect, please try again" message.

Reply:
You're right, that would be much better!